packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -19,6 +19,7 @@ import {
 const log = new Logger({
   name: 'oe:batch-submitter:init',
   sentryOptions: {
+    release: `@eth-optimism/batch-submitter@${process.env.npm_package_version}`,
```

Comment:
Is this injected in at runtime by `yarn`?

Reply:
Tested locally like this:

```bash
❯ cat exec/run-batch-submitter.js
#!/usr/bin/env node

console.log(process.env.npm_package_version);
process.exit(1)

const batchSubmitter = require("../dist/src/exec/run-batch-submitter")

batchSubmitter.run()

❯ yarn start
yarn run v1.22.10
$ node ./exec/run-batch-submitter.js
0.1.12
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
This means that `yarn` or `npm` MUST be the entrypoint or it will be undefined